ludwig/backend/horovod.py

```diff
@@ -37,7 +38,8 @@ def initialize_pytorch(self, *args, **kwargs):
         initialize_pytorch(*args, horovod=self._horovod, **kwargs)
 
     def create_trainer(self, **kwargs):
-        return Trainer(horovod=self._horovod, **kwargs)
+        config, kwargs = load_config_with_kwargs(Trainer.get_schema_cls(), kwargs)
```

Comment:
Seems redundant to put this in the `create_trainer` Backend implementation. Can we just move this to `api.py` and pass it in as one of the kwargs?

Reply:
moved